### DISCUSSION TITLE
Saved Mappings in MeshCentral Router not loading

### DISCUSSION THREAD

**Discussion by jrnoble83** [Ylianst/MeshCentral#4036] [General]
Hello,  I am running 2 instances of MeshCentral v 1.0.27 with MeshCentral Router v1.8.8171.  My issue is that mappings I have tried to save using the save mapping feature save to a file, however when I try to open it nothing happens.  Is anyone else experiencing this issue?

**Comment by Ylianst** [comment]
I put in a fix for this, #4033. Anyway you can confirm it's fixed and close the issue? - Thanks.
